exporter/cassandraexporter/exporter_traces.go

```diff
@@ -24,6 +24,12 @@ type tracesExporter struct {
 
 func newTracesExporter(logger *zap.Logger, cfg *Config) (*tracesExporter, error) {
 	cluster := gocql.NewCluster(cfg.DSN)
+	if cfg.Auth.UserName != "" && cfg.Auth.Password != "" {
```

Comment:
It would be good to add testing for this change to make sure the username and password are handled properly.

Reply:
I agree with it. Unfortunately, creating a cluster doesn't return an interface, so this is difficult to mock. I made a small refactoring so that the authorization configuration can be checked.
I moved the cluster initialization into a separate function and tested it.

Reply:
Totally fair. Just wanted to make sure the arguments are getting passed around properly, and your tests check the basics of what I was hoping for. Thank you!